tests/typedefs_a_test.py

```
@@ -0,0 +1,9 @@
+import pytest
+import pytypedefs_a as m
+import pytypedefs_b as m_b
+
+def test_alias():
```

We could also test for a raise in case of a missing import?